pom.xml

```diff
@@ -464,8 +467,9 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
+          <source>8</source>
```

Comment:
@hiteshbosamiya Is `source` and `target` really `8` instead of `1.8` ?
Everything I found was with `1.8`, like here: https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-source-and-target.html

Reply:
8 is synonym for 1.8 (https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javac.html). Just to be consistent for all the 3 attributes, I changed source and target also 8. For all the major versions i.e. starting from 1.5, we can use single digit.

Reply:
Thanks for the link 👍